functionary/ui/templates/partials/environments/environment_add_user.html

```diff
@@ -0,0 +1 @@
+{% include "partials/modal.html" with title="Add user" description="Add user to environment." content="forms/environments/environment_add_user.html" %}
```

Comment:
Can we leverage the Django template paradigm better using {% block %}'s for overrides vs passing all this as parameters?